addons/sale/models/sale_order_line.py

```diff
@@ -1651,19 +1651,24 @@ def _get_product_catalog_lines_data(self, **kwargs):
             {
                 'quantity': float,
                 'price': float,
+                'productUnitPrice': float (optional)
                 'readOnly': bool,
                 'uomDisplayName': String,
+                'productUomDisplayName': string
             }
         """
         if len(self) == 1:
+            price = self._get_discounted_price()
             return {
                 'quantity': self.product_uom_qty,
-                'price': self._get_discounted_price(),
+                'price': price,
+                'productUnitPrice': self.product_uom_id._compute_price(price, self.product_id.uom_id),
                 'readOnly': (
                     self.order_id._is_readonly()
                     or bool(self.combo_item_id)
                 ),
                 'uomDisplayName': self.product_uom_id.display_name,
+                'productUomDisplayName': self.product_id.uom_id.display_name,
```

Comment:
Since this is a static info on the product, can't it be managed by the kanban view and the product data instead ?

Seems strange that we have to send this info through the order/line info when it's static and related to the product only 🤔